frontend/src/stores/ParticipantStore.js

```diff
@@ -108,6 +117,18 @@ export class ParticipantStore {
     this.routeToQueueTable = data
   }
   // Participant State Actions
+  @action handleParticipantChange = event => {
+    switch (event.target.name) {
```

Comment:
Switch is ok here because these forms will not be changing that much over time. We will have to figure out a different way to do transformations on the 'data' forms (if at all). This is a thinking out loud comment, not a request for change

Reply:
One issue here is that radio buttons return a string value and we need a boolean.

Reply:
Yea totally. I thought about asking for a custom method in ParticipantInfo that handles this and keeping the handleChange without logic, but that's not really scalable for dynamic forms either.  

The "data" forms are purposely going to rely heavily on Selects for data integrity, and we can handle cascading effects in useEffect hooks (or similar) instead of the on change method.  Again, just thinking out loud